upgrade/php/ps_800_update_tab_lang.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+/**
+ * Updates ps_tab_lang table for a given domain and className
+ *
+ * This method will fetch the tab from className, and update ps_tab_lang
+ * with translated wordings for all available languages
+ *
+ * @param string $domain
+ * @param string $className
+ */
+function ps_800_update_tab_lang($domain, $className)
```

Comment:
```suggestion
function ps_update_tab_lang($domain, $className)
```

Reply:
Since it's meant to be a tool function that can be reused for other versions (even older ones maybe if we find bugs) I'd suggest a more generic name for this one